src/Resources/config/routes.yaml

```
@@ -7,3 +7,9 @@ setono_sylius_cms_shop:
 setono_sylius_cms_admin:
     resource: "@SetonoSyliusCMSPlugin/Resources/config/routes/admin.yaml"
     prefix: /admin/cms
+
+setono_sylius_cms_download_asset:
+    path: /cms/asset/{id}-{filename}
```

```suggestion
    path: /cms/asset/{id}
```

This will allow us to make better content negotiation also. Let's say the filename is `image.jpg`, but we want to show an optimized `image.webp`